src/components/Note/Note.jsx

```diff
@@ -1,34 +1,58 @@
 import React from "react";
-import { useNotes } from "../../context/notes-context";
 import "./note.css";
+import { useNotes } from "../../context/notes-context";
 import { FiTrash2 } from "react-icons/fi";
 import { RiInboxArchiveLine } from "react-icons/ri";
-import { MdOutlineColorLens } from "react-icons/md";
+import { BsFillPinFill, BsPin } from "react-icons/bs";
```

Comment:
Suggestion : You can have icons.js file and then import these icons from there.

Reply:
nice idea